### DISCUSSION TITLE
2024-08-06 화요일

### DISCUSSION THREAD

**Discussion by eunohhh** [nbc-final-a10/TripBuddy#207] [기록보드]
2024-08-06 화요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 아침 회의 | |
| 10:00 ~ 11:00 | 집중 개발 time | |
| 11:00 ~ 13:00 | 집중 개발 time | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 16:00 | MVP 발표 | |
| 16:00 ~ 18:00 | 집중 개발 TIME | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 저녁 회의 | |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

```
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- 내용

### 트러블 슈팅

- 내용

### 내일 계획

- 내용
```

```
// 이미지 테이블
<table>
  <tr>
    <th>이미지 1번 제목</th>
    <th>이미지 2번 제목</th>
  </tr>
  <tr>
    <td><img src="이미지 주소" width="300" height="300"/></td>
    <td><img src="이미지 주소" width="300" height="300"/></td>
  </tr>
</table>
```

**Comment by eunohhh** [comment]
## 개인 기록보드 서식 (오은)

### 오늘 한 일

- 미들웨어에서 쿠키 사용하여 첫 접속이면 튜토리얼로 가게끔 처리...힘들다
- 여정 카드 1/4 인원수 표시부분 contract 에서 가져와서 카운트 보여지게끔 처리
- 스토리 작성완료시 해당 스토리 페이지로 리다이렉트

### 트러블 슈팅

- 수파베이스 auth 와 함께 미들웨어 사용시, 커스텀 할 때 주의해야 합니다. 살짝만 로직이 잘못 작성되어도 예상치 못한 결과가 발생합니다.
- 미들웨어에서 어디론가 리다이렉트 시키는 로직을 작성할 경우 if 문에 startsWith('/api') 를 꼭 넣어야 합니다! (체크할때마다)

### 내일 계획

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- funnel button 커스텀 훅 중복 통신 방지 코드 리팩토링
- 글쓰기 페이지 전체 컴포넌트 유효성 검사

### 트러블 슈팅

- 와이어프레임 상 '글쓰기' 실패 로직이 없기 때문에 실패 처리에 대해서 고민이 됨.
- 퍼널에서는 무조건 다음 스텝으로 넘어가고 보기 때문에 의사코드가 쉽게 떠오르지 않음.
  - 7, 8 스텝 등으로 구분해서... 통신에 성공할 경우 step을 7로 강제로 넘기거나... 실패 시 8로 넘기거나...
  - 아니면 통신 성공 여부를 지켜보고 다음 스텝으로 증가시키고 실패하는 경우 스텝 증가를 시키지 않거나...

### 내일 계획

- 리치텍스트 에디터 구현

**Comment by slsl2** [comment]
## 개인 기록보드 서식 (김민지)

### 오늘 한 일

- chat 페이지 리팩토링 중

### 내일 계획

- chat 페이지 리팩토링
- 채팅 목록 페이지 메시지 쌓이는 숫자 표시

**Comment by hyunjeongkwak** [comment]
#### 오늘 한 일

- 위치, 날짜 페이지 나누는 중

#### 내일 계획

- 페이지 나누기 완료
- pc 버전 UI 작업
